husky_description/urdf/husky.urdf.xacro

```diff
@@ -141,14 +218,14 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
 
     </xacro:if>
 
-    <xacro:if value="$(arg laser_enabled)">
+    <xacro:if value="$(optenv HUSKY_LASER_ENABLED false)">
 
       <sick_lms1xx_mount prefix="base" topic="scan"/>
 
       <joint name="laser_mount_joint" type="fixed">
         <origin xyz="$(optenv HUSKY_LMS1XX_XYZ 0.2206 0.0 0.00635)"
                 rpy="$(optenv HUSKY_LMS1XX_RPY 0.0 0.0 0.0)" />
-        <parent link="top_plate_link" />
+        <parent link="$(optenv HUSKY_LASER_PARENT_LINK top_plate_link)" />
```

Comment:
@tonybaltovski @paulbovbel 

With the addition of the bulk-head, I figured I needed to be able to easily change the parent link of the laser due to the structure that the payloads sit on changing all the time